proto/ibc/applications/middleware/fee/v1/fee.proto

```diff
@@ -0,0 +1,21 @@
+syntax = "proto3";
+
+package ibc.applications.middleware.fee.v1;
+import "cosmos/base/v1beta1/coin.proto";
+import "gogoproto/gogo.proto";
+import "ibc/core/channel/v1/channel.proto";
+option go_package = "github.com/cosmos/ibc-go/modules/apps/29-fee/types";
+
+// Fee interface
+// See Fee Payment Middleware spec:
+// https://github.com/cosmos/ibc/tree/master/spec/app/ics-029-fee-payment#fee-middleware-contract
+message Fee {
+  repeated cosmos.base.v1beta1.Coin amount = 1
+      [(gogoproto.nullable) = false, (gogoproto.castrepeated) = "github.com/cosmos/cosmos-sdk/types.Coins"];
+}
+
+// Fee associated with a packet_id
+message IdentifiedPacketFee {
```

Comment:
We need to be able to determine which relay type the fee corresponds to. There's two approaches, define a type with:
```
  ibc.core.channel.v1.PacketId           packet_id   = 1 [(gogoproto.moretags) = "yaml:\"packet_id\""];
  ibc.applications.middleware.fee.v1.Fee receive_fee = 2 [(gogoproto.moretags) = "yaml:\"receive_fee\""];
  ibc.applications.middleware.fee.v1.Fee ack_fee     = 3 [(gogoproto.moretags) = "yaml:\"ack_fee\""];
  ibc.applications.middleware.fee.v1.Fee timeout_fee = 4 [(gogoproto.moretags) = "yaml:\"timeout_fee\""];
  repeated string                        relayers    = 5;
  ```
 as MsgEscrowFee does now, or create individual mappings as the rpc's do now (different endpoint for recv, ack, timeout)
 
 I don't see a strong design reason not to just bundle all 3 fee's into one type. Thoughts @AdityaSripal @seantking

Reply:
Yea I think its fine to store them in state together as 1 struct containing all 3 fees. I'd still like to have the different rpcs, and have the query funcs split out each fee and return the relevant one to relayer. This would at least conform to spec as is.

Making it so we only send back bundle is a difference from the spec. We could do it, but we would need to make the API changes on spec level as well.

Reply:
Lets do both for now and ask relayers which they'd prefer?

Reply:
I updated the `IdentifiedPacketFee` structure. 

Just so we're clear, we are also saying that we want three new queries for getting all of a specific type of fee (timeout etc)?

Reply:
Added another query as requested